doc/faq.rst

```diff
@@ -0,0 +1,229 @@
+:orphan:
+
+.. _manual:
+
+===
+FAQ
+===
+
+Errors
+======
+
+Auto-sklearn is extremely memory hungry in a sequential setting
```

Comment:
Hm, is this really an error? How about calling it "issues"?

Reply:
Good idea, I'll change that.